scripts/build/build-chart-repo.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+echo '>> Building charts...'
+find "$HELM_CHARTS_SOURCE" -mindepth 1 -maxdepth 1 -type d | while read -r chart; do
```

Comment:
After the merge of `prereqs/` and `operator/`, there will be only single directory left, right? So maybe we could then remove the `find` & `while`, and instead just set:

    chart="$HELM_CHARTS_SOURCE/eunomia-operator"

?